nodes/object/object_name_filter.py

```diff
@@ -0,0 +1,54 @@
+import bpy
+from ... events import executionCodeChanged
+from ... base_types.node import AnimationNode
+
+class ObjectNameFilterNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ObjectNameFilterNode"
+    bl_label = "Object Name Filter"
+    
+    def checkedPropertiesChanged(self, context):
+        executionCodeChanged()
+    
+    useCaseSensitive = bpy.props.BoolProperty(default = False, description = "Use Case Sensitive", update = checkedPropertiesChanged)
+    
+    filterType = bpy.props.EnumProperty(items = [("STARTS_WITH", "Starts With", "All Objects with names starting with"),
+                                                 ("ENDS_WITH", "Ends With", "All Objects with names ending with")],
```

Comment:
can you maybe define the items above the class?
the variable containing the items should be called *filterTypeItems*